RELEASES.md

```diff
@@ -77,6 +77,24 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
         
     c. Create a PR on github into the release branch. Ask in the UA channel on mattermost for review.
 
+    d. When reviewing the release PR, please use the following guidelines when reviewing the new changelog entry:
+
+       * Is the version correctly updated ? We must ensure that the new version on the changelog is
+         correct and it also targets the latest Ubuntu release at the moment.
+       * Is the entry useful for the user ? The changelog entries should be user focused, meaning
```

Comment:
This one I am not entirely sure if everyone agrees. I will let the team expresses their opinion about it

Reply:
I agree with this one.
Of course there will be changelog entries related to internals and such, but even those need to be written targeting the user. 'user' here might be just a regular user, but also product/project managers, other engineers, etc, so we can group small bug fixes and such into a single "fixes" entry.

Reply:
I also agree!

Reply:
Yes, let's spend more time pruning our logs, but some of that motivation is to give credit to community members who have contributed, and reflect that ownership more publicly. Since UA is mostly internal to canonical and we are the primary devs, we don't need that level of attribution. We should distill logs to functional/feature-related fixes that make the most sense for what was released.